Formula/unionfs-fuse.rb

```diff
@@ -0,0 +1,22 @@
+class UnionfsFuse < Formula
+  desc "FUSE based union filesystem with COW"
+  homepage "https://github.com/rpodgorny/unionfs-fuse"
+  url "https://github.com/rpodgorny/unionfs-fuse/archive/v2.1.tar.gz"
+  sha256 "c705072a33a18cbc73ffe799331d43410b6deef5d6f2042038f8fd3ab17b6e2e"
+  head "https://github.com/rpodgorny/unionfs-fuse.git"
+
+  depends_on "pkg-config" => :build
+  depends_on :osxfuse
+
+  def install
+    system "make"
+    bin.install "src/unionfs"
```

Comment:
Is there no `make install`?

Reply:
The makefile has a fixed prefix of `/usr/local` and has no configure script so when I try just putting `system "make", "install"` in the Formula it fails with error 71 after not being able to install the files. Manually telling Homebrew to install the files seems to have fixed the issue.

Reply:
You should submit a patch upstream to allow using `DESTDIR` then